.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
```

Comment:
Because now the 2/3 split is handled by tox, below. 

We can get the best of both worlds with tox-travis: https://pypi.python.org/pypi/tox-travis/0.4